django_server/settings.py

```diff
@@ -167,6 +167,8 @@
 }
 
 SITE_ID = 1
+if(DEBUG):
```

Comment:
Why the parenthesis?

Reply:
just my way of writing code :smile: .I can change if you want to.

Reply:
Yes please change :smile: